Packages/com.unity.inputsystem/InputSystem/Controls/TouchFlagsControl.cs

```diff
@@ -4,23 +4,23 @@
 
 namespace UnityEngine.Experimental.Input.Controls
 {
-    public class TouchTypeControl : InputControl<TouchType>
+    public class TouchFlagsControl : InputControl<TouchFlags>
```

Comment:
I wonder whether we shouldn't nuke this entirely. With this now being a bitfield, we could simply treat it as buttons.

The pen, for example, has an "in-range" bit. It's simply exposed as a button on the pen. Wondering whether we shouldn't just do the same thing here.